src/api/mock-interceptor.ts

```diff
@@ -36,22 +38,22 @@ export const interceptRequestsOnMock = () => {
   const mock = new MockAdapter(Axios);
 
   // SEARCH
-  mock.onGet(new RegExp(`/${ApiBaseUrl.PUBLICATIONS}/*`)).reply(200, mockPublications);
+  mock.onGet(new RegExp(`${PublicationsApiPaths.SEARCH}/*`)).reply(200, mockPublications);
 
   // Create publication from doi
-  mock.onPost(new RegExp(`/${ApiBaseUrl.PUBLICATIONS}/doi/*`)).reply(200, mockDoiPublication);
+  mock.onPost(new RegExp(`${PublicationsApiPaths.DOI}/*`)).reply(200, mockDoiPublication);
 
   // lookup DOI
-  mock.onGet(new RegExp(`/${ApiBaseUrl.DOI_LOOKUP}/*`)).reply(200, mockDoiLookupResponse);
+  mock.onGet(new RegExp(`${PublicationsApiPaths.DOI_LOOKUP}/*`)).reply(200, mockDoiLookupResponse);
 
-  // CRISTIN
-  mock.onGet(new RegExp(`${CRISTIN_API_URL}/projects*`)).reply(200, mockCristinProjects, { 'X-Total-Count': '12' });
+  // PROJECT
+  mock.onGet(new RegExp(`${ProjectsApiPaths.PROJECTS}/*`)).reply(200, mockProjects);
 
   // PUBLICATION CHANNEL
   mock.onPost(new RegExp(`${API_URL}${PublicationChannelApiPaths.SEARCH}`)).reply(200, mockNsdPublisers);
 
   // USER
-  mock.onGet(new RegExp(`/${ApiBaseUrl.USER}/*`)).reply(200, mockUser);
+  // mock.onGet(new RegExp(`/${ApiServiceUrl.USER}/*`)).reply(200, mockUser);
```

Comment:
hvordan blir denne nå ?

Reply:
vi må ha med denne. hvis ikke så får vi vel ikke opp noen brukerinfo?

Reply:
eller er den ikke brukt?

Reply:
delete this shit